### PR TITLE
Add method getTicketBySector

### PR DESCRIPTION
**Pull request by vrusalovskaya**
task JFB-2

### CODE REVIEW

**Comment by stvjbs** [comment]
great work. Will test it tomorrow

**Review by deolexx — APPROVED** [review]
LGTM

File: src/main/java/com/andersenlab/lesson2/TicketService.java [hunk 1]
```diff
@@ -19,5 +20,20 @@ public static void main(String[] args) {
...
+        System.out.println(getTicketBySector(StadiumSector.B));
+    }
+
+    private static Ticket getTicketBySector (StadiumSector sector){
+        for (Ticket ticket : storage) {
+            if (ticket.getStadiumSector() == sector){
+                return ticket;
+            }
+        }
+        return null;
```
Comment (deolexx) [inline src/main/java/com/andersenlab/lesson2/TicketService.java:37]: Return null from method like we have here it is not a best practice. It is recommended to use alternatives like Optional (in Java 8 and above) or throwing exceptions where appropriate.

**Review by deolexx — APPROVED** [review]
LGTM

**Pull request closed without merging**